docs/technical-design/database-diagram.md

```diff
@@ -76,3 +76,83 @@ Question ||--o{ QuestionReponse: questionID
 QuestionReponse ||--o{ QuestionResponseDocument: questionResponseID
 User  }o--o{  State : ""
 ```
+
+
+## Contract & Rates DB
+
+This is the db diagram for the contracts and rates work, still disconnected from the rest of the db. 
+
+```mermaid
+erDiagram
+
+ContractTable {
+   String id
+}
+ContractRevisionTable {
+   String id
+   String contractID
+   DateTime createdAt
+   DateTime updatedAt
+
+   String unlockInfoID
+   String submitInfoID
+
+   String name
+}
+
+RateTable {
+   String id
+}
+RateRevisionTable {
+   String id
+   String rateID
+   DateTime createdAt
+   DateTime updatedAt
+
+   String unlockInfoID
+   String submitInfoID
+
+   String name
+}
+
+RateRevisionsOnContractRevisionsTable {
+   String rateRevisionID
+   String contractRevisionID
+
+   DateTime validAfter
+   DateTime validUntil
+   
```

Comment:
Make sure `isRemoval` goes here - or else we not add this now at all and instead add when we publish docs